File: src/test/java/edu/hm/hafner/analysis/assertj/IssueSoftAssertions.java
package edu.hm.hafner.analysis.assertj;

import org.assertj.core.api.SoftAssertions;

import edu.hm.hafner.analysis.Issue;
import edu.hm.hafner.analysis.Issues;

public class IssueSoftAssertions extends SoftAssertions {

    public IssueAssert assertThat(final Issue actual) {
        return proxy(IssueAssert.class, Issue.class, actual);
    }

}

File: src/test/java/edu/hm/hafner/analysis/assertj/kotlin/IssueSoftAssertions.kt
package edu.hm.hafner.analysis.assertj

import edu.hm.hafner.analysis.Issue

open class IssueSoftAssertions : SoftAssertions() {
    fun assertThat(actual: Issue): IssueAssert {
        return proxy(IssueAssert::class.java, Issue::class.java, actual)
    }
}
